community/modules/scripts/htcondor-install/files/install-htcondor-autoscaler-deps.yml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+- name: Install but do not activate HTCondor autoscaler
+  become: true
+  hosts: localhost
+  tasks:
+  - name: Install Python 3 pip
+    ansible.builtin.yum:
```

Comment:
Will this always be run on centos/redhat images? If not, ansible.buildtin.package would be a more flexible choice here.

Reply:
I will check the package names on Debian derivatives. If so, I will make the change. There are all sorts of CentOS 7 assumptions here and throughout but we should not be adding more.

Reply:
Resolved on HEAD of PR